### ISSUE TITLE
AttributeError: module 'gridencoder' has no attribute 'grid_encode_forward'

### ISSUE THREAD

**Issue by SpicyMelonYT** [ashawkey/stable-dreamfusion#87]
AttributeError: module 'gridencoder' has no attribute 'grid_encode_forward'

to get it working I had to remove the _ from the _gridencoder. In other AI programs with colab, this worked fine. But later down this colab, this error popped up and it might be from the name change but I am not sure. Is this a known error, I couldn't find it.

**Comment by SpicyMelonYT** [comment]
Just giving a ping! I would love an answer to this thank you!

**Comment by Castrol68** [comment]
Hello, I have the same problem, and how do you deal with it?

**Comment by SpicyMelonYT** [comment]
No clue sorry, I never received help and stopped trying. Try asking Bing Chat Or Chatgpt!

**Comment by luis-guerraf** [comment]
Any news on this?

**Comment by SpicyMelonYT** [comment]
unfortunately no, i think this issue is burried under so much. Maybe try making another issue about it and it will cause all the "post duplicate" nazi's to show up and maybe help LOL

**Comment by luis-guerraf** [comment]
haha I fixed it on my end at least. I had replaced `import _gridencoder` with `gridencoder` (no underscore) following this advice https://github.com/eladrich/latent-nerf/issues/7#issuecomment-1416687218. I guess the name should match the name assigned to the CUDAExtension, rather than the package, in setup.py.

**Comment by SpicyMelonYT** [comment]
yeah taking out the underscores are usually how to do it. But i tried that a long time ago, didn't fix it for me unfortunately!

**Comment by ruiqixu37** [comment]
I fixed it by downgrading my CUDA version from 12.1 to 11.7, and then re-building the extensions. I am running on Windows without WSL.
